galaxy_ng/app/api/v3/serializers/task.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+from rest_framework import serializers
+
+from galaxy_ng.app.api.ui.serializers.base import Serializer
+
+log = logging.getLogger(__name__)
+
+
+class TaskSerializer(Serializer):
```

Comment:
Not sure if subclassing the pulpcore serializer is needed here, so keeping the fields simple for what is needed now